src/components/button/Button.tsx

```diff
@@ -112,207 +117,239 @@ const AminoButton = styled.button<ButtonProps<GroupTag>>`
+      ? `border: 1px solid ${theme.orange300}; border-radius: ${theme.radius6};`
+      : ''}
 
   &:not([disabled]) {
     &:hover {
-      background: linear-gradient(
-        180deg,
-        ${theme.gray0} 12.5%,
-        ${theme.gray100} 87.5%
-      );
-      background: ${p => p.hoverBackground && getAminoColor(p.hoverBackground)};
-      border: 1px solid ${theme.gray200};
+      background: ${p => {
+        if (p.hoverBackground) {
+          return p.hoverBackground;
+        }
+        if (p.outline) {
+          return theme.orange100;
+        }
+        return theme.buttonWarningHover;
+      }};
     }
     &:active {
-      background: ${theme.blue100};
-      color: ${theme.blue600};
-      border: 1px solid ${theme.blue300};
+      background: ${p => (p.outline ? theme.orange200 : theme.orange700)};
     }
   }
 
+  &[disabled] {
+    background: ${p => (p.outline ? theme.gray0 : theme.orange400)};
+  }
+
   ${StyledSpinnerWrapper} {
-    background: ${theme.gray0};
+    background: ${p => (p.outline ? theme.gray0 : theme.orange600)};
   }
 `;
 
 const Subtle = styled(AminoButton)`
   background: ${p => getAminoColor(p.background) || 'none'};
-  color: ${p => getAminoColor(p.color) || theme.gray800};
+  color: ${p => getAminoColor(p.color) || theme.textColorSecondary};
 
   &:not([disabled]) {
     &:hover {
       background: ${p => getAminoColor(p.hoverBackground) || theme.gray100};
     }
     &:active {
-      background: ${theme.blue100};
-      color: ${theme.blue600};
+      background: ${theme.gray200};
     }
   }
 
   &.loading {
     color: transparent;
   }
-`;
-
-const TextButton = styled(AminoButton)<ButtonProps<GroupTag>>`
-  color: ${p => getAminoColor(p.color) || theme.gray800};
-  height: 20px;
-  line-height: 20px;
 
   &[disabled] {
-    color: ${theme.gray400};
-    &:not(.loading) {
-      opacity: inherit;
+    .content {
+      opacity: 0.6;
     }
   }
-
-  &:not([disabled]) {
-    &:hover {
-      color: ${theme.gray700};
-    }
-    &:active {
-      color: ${theme.gray1200};
-    }
-  }
-
-  &.loading {
-    color: transparent;
-  }
 `;
 
 const PlainButton = AminoButton;
 
 const LinkButton = styled(AminoButton)<ButtonProps<GroupTag>>`
-  color: ${p => getAminoColor(p.color) || theme.blue600};
+  color: ${p => getAminoColor(p.color) || theme.primary};
 
   &:not([disabled]) {
     &:hover {
-      background: ${p => getAminoColor(p.hoverBackground) || theme.gray100};
+      background: ${p => getAminoColor(p.hoverBackground) || theme.blue100};
     }
     &:active {
-      background: ${theme.blue100};
-      color: ${theme.blue800};
+      background: ${theme.blue200};
     }
   }
 
-  ${StyledSpinnerWrapper} {
-    background: ${theme.gray0};
+  &.loading {
+    color: transparent;
+  }
+
+  &[disabled] {
+    .content {
+      opacity: 0.6;
+    }
   }
 `;
 
-type IntentProps = 'outline' | 'subtle' | 'text' | 'link' | 'plain' | Intent;
+type IntentProps = 'subtle' | 'link' | 'plain' | Intent;
```

Comment:
If we are using this opportunity to complete change the new button API, then I would prefer this prop be called `variant`